tests/code_size/hello_world_wasm.json

```diff
@@ -1,10 +1,10 @@
 {
   "a.html": 665,
   "a.html.gz": 427,
-  "a.js": 322,
-  "a.js.gz": 259,
+  "a.js": 338,
+  "a.js.gz": 269,
   "a.wasm": 104,
   "a.wasm.gz": 112,
-  "total": 1091,
-  "total_gz": 798
+  "total": 1107,
+  "total_gz": 808
```

Comment:
A good rule of thumb is that new PRs are not allowed to regress the generated code size.

Reply:
Reversed